tests/lib/create_tpp_test_db.py

```diff
@@ -0,0 +1,31 @@
+"""
+Run this using:
+
+    pytest -o python_functions=create tests/lib/create_tpp_test_db.py
```

Comment:
this is different from what's in the [justfile](https://github.com/opensafely-core/ehrql/blob/77b644437f007e5d7ce36d0c595271372ed97e1f/Justfile#L153) is there a meaningful difference between `pytest` and `python -m pytest` here?

Reply:
Nope, other than `pytest` is what I'd use myself from the command line but I went with `python -m pytest` in the Justfile for consistency with the rest of the Justfile. Either's fine.